ocp_resources/mtv.py

```diff
@@ -71,7 +71,7 @@ def wait_for_resource_status(
         condition_message=None,
         condition_reason=None,
         condition_category=None,
-        wait_timeout=600,
+        wait_timeout=1200,
```

Comment:
please send 1200 when you call the function, no need to change the default.

Reply:
sorry, did not mean to send this PR